packages/security/test/config/mobile-services.json

```diff
@@ -24,6 +24,14 @@
       "type": "push",
       "url": "https://www.mocky.io/v2/5a6b59fb31000088191b8ac6",
       "config": {}
+    },
+    {
+      "id": "security",
```

Comment:
should this is `security` or `appsecurity`?

Reply:
Think `security` is fine as future plans involve bringing the deviceTrust into the mobile security service